apiserver/backrestservice/backrestimpl.go

```diff
@@ -593,3 +623,18 @@ func createRestoreWorkflowTask(clusterName, ns string) (string, error) {
 	}
 	return spec.Parameters[crv1.PgtaskWorkflowID], err
 }
+
+// clusterNamesToPGClusterList takes a list of cluster names as specified by a slice of
+// strings containing cluster names and then returns a PgclusterList containing Pgcluster's
+// corresponding to those names.
+func clusterNamesToPGClusterList(clusterNames []string, namespace string) (crv1.PgclusterList,
+	error) {
+	selector := fmt.Sprintf("%s in(%s)", config.LABEL_NAME, strings.Join(clusterNames, ","))
+	clusterList := crv1.PgclusterList{}
+	err := kubeapi.GetpgclustersBySelector(apiserver.RESTClient, &clusterList, selector,
+		namespace)
+	if err != nil {
```

Comment:
A place for your favorite syntactical expression :wink:

Reply:
Syntax updated.